doc/rst/language/spec/memory-consistency-model.rst

```diff
@@ -322,8 +322,8 @@ section:
    memory.
 
 The *unordered* loads and stores :math:`UL(a)` and :math:`US(a)` respect
-fences but not program order. As in
-Section \ `31.1.2 <#memory_order>`__, unordered loads and stores are
+fences but not program order. As in :ref:`memory_order`,
```

Comment:
This one seems easy to misread; I think it needs to say "as in the section" or something along the lines; otherwise people might think it's referring to the language element `memory_order`.